study.md

```diff
@@ -160,15 +169,42 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+response is the answer to the request.
+
+the response contains the status code of the request, header information and body
+of the request.
+
+200 OK
+This response code indicates that the request was successful.
+
+201 Created
+This indicates the request was successful and a resource was created. It is used to confirm success of a PUT or POST request.
+
+400 Bad Request
+The request was malformed. This happens especially with POST and PUT requests, when the data does not pass validation, or is in the wrong format.
+
+404 Not Found
+This response indicates that the required resource could not be found. This is generally returned to all requests which point to a URL with no corresponding resource.
+
+401 Unauthorized
+This error indicates that you need to perform authentication before accessing the resource.
+
+405 Method Not Allowed
+The HTTP method used is not supported for this resource.
+
+409 Conflict
+This indicates a conflict. For instance, you are using a PUT request to create the same resource twice.
+
+500 Internal Server Error
+When all else fails; generally, a 500 response is used when processing fails due to unanticipated circumstances on the server side, which causes the server to error out.
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl https://www.reddit.com
```

Comment:
Make sure you use the request flag and the type of request (GET, DELETE, etc).

`curl --request GET https://www.reddit.com`